lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -439,7 +443,7 @@ describe('LMSFilePicker', () => {
 
   it('enables "Select" button when a file is selected', async () => {
     const wrapper = renderFilePicker();
-    await fakeApiCall;
+    await waitFor(() => fakeApiCall.called);
```

Comment:
Could change this to `await waitForElement('FileList')` given the code that follows.